gordo_components/cli/cli.py

```diff
@@ -89,6 +89,12 @@ def gordo():
     "multiple times. Separate key,valye by a comma. ie: --model-parameter key,val "
     "--model-parameter some_key,some_value",
 )
+@click.option(
+    "--model-location-file",
+    help="Full path to a file to create and write the location of where the serialized model is placed.",
+    type=click.File(mode="w"),
```

Comment:
I think maybe you want `lazy=False` to force any errors to occur early (before model build)